src/site/content/en/metrics/lcp/index.md

```diff
@@ -43,16 +47,13 @@ Performance Working Group](https://www.w3.org/webperf/) and research done at
 Google, we've found that a more accurate way to measure when the main content
```

Comment:
This is a bit misleading as it makes it sound like LCP is a metric approved by the WebPerf WG. It's not. Perhaps we should limit this to research done at Google, and say it's incubated under the WICG.

Reply:
This was the original language we bikeshedded with Ilya and Yoav when the original post was published, so I think I'll leave it for now. But I do think it's less relevant in the new format, so I assume we'll drop it eventually.